device-types/Dell/PowerScale-F710.yaml

```diff
@@ -0,0 +1,42 @@
+---
+manufacturer: Dell
+model: PowerScale F710
+slug: dell-powerscale-f710
+comments: '[Dell PowerScale F710 Specification Sheet](https://www.delltechnologies.com/asset/en-us/products/storage/technical-support/h15963-ss-powerscale-all-flash-nodes.pdf)'
+description: All-Flash NAS Platform
+u_height: 1
+is_full_depth: true
+airflow: front-to-rear
+front_image: true
+rear_image: true
+weight: 22.5
+weight_unit: kg
+console-ports:
+  - name: Console
+    type: de-9
+power-ports:
+  - name: PS1
```

Comment:
Thank you for contribution. As these are removable PSUs they should be modeled as `module-bays` and then add the required PSU modules in the modules folder if they don't already exist.